staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -63,10 +64,10 @@ const (
 )
 
 var supportedValidationReason = sets.NewString(
-	string(field.ErrorTypeInvalid),
-	string(field.ErrorTypeForbidden),
-	string(field.ErrorTypeRequired),
-	string(field.ErrorTypeDuplicate),
+	string(metav1.StatusReasonForbidden),
+	string(metav1.StatusReasonInvalid),
+	string(metav1.StatusReasonAlreadyExists),
+	string(metav1.StatusReasonNotFound),
```

Comment:
we should keep the same values as before, since these get returned as field errors